meerk40t/grbl/plugin.py

```diff
@@ -175,6 +170,36 @@ def plugin(kernel, lifecycle=None):
                 ],
             },
         )
+        kernel.register(
+            "dev_info/grbl-longer-ray5",
+            {
+                "provider": "provider/device/grbl",
+                "friendly_name": _("Longer Ray5 (GRBL)"),
+                "extended_info": _("Longer-branded 5w/10w/20w grbl diode laser.\nMake sure you verify your bed size! This machine has several upgrade kits."),
+                "priority": 21,
```

Comment:
That should be removed, shouldn't it?

Reply:
If you're talking about the preset for the Longer Ray5, no, i had intentionally added it as a preset. It's basic but helps people getting set up, some might not necessarily know what "GRBL" means in the context of laser engravers, and will assume their device isn't supported. (Example being my business partner)

Do you personally think it's a bad idea to include presets?